src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx

```diff
@@ -46,13 +46,9 @@ Keep reading for details about New Relic One pricing and billing.
 
 ## Billing and usage in the UI [#billing-usage-ui]
 
-The ability to edit billing-related settings depends on user permissions. For users on the New Relic One user model, this requires the [**Billing user**](https://docs.newrelic.com/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure#roles) role. For users on the original user model, this requires an [Owner or Admin role](https://docs.newrelic.com/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model#roles). 
+The ability to edit billing-related settings depends on user permissions. For users on the New Relic One user model, this requires the [**Billing user**](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure#roles) role. For users on the original user model, this requires an [Owner or Admin role](/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model#roles). 
```

Comment:
I'm trying to think of a way to make that first sentence a little less passive... How about: "Your user role permissions determine whether you have the ability to edit billing-related settings." What do you think?